github/Organization.py

```diff
@@ -60,6 +60,20 @@ class Organization(github.GithubObject.CompletableGithubObject):
     This class represents Organizations. The reference can be found here http://developer.github.com/v3/orgs/
     """
 
+    def __init__(self, requester, headers, attributes, completed):
+        attrs = {}
+
+        # GitHub currently returns invalid URLs for organizations.
+        # Work around this by replacing with the correct URLs.
+        right = "https://api.github.com/orgs"
+        wrong = "https://api.github.com/users"
+        for k, v in attributes.items():
+            if isinstance(v, str):
+                v = v.replace(wrong, right)
```

Comment:
Just .replace('/users', '/orgs') -- we can't rely on the URL. Part of me was trying to concoct a dict comprehension for this, but we're limiting by only replacing for strings. :-(